routes/web.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * Because these routes should mimic static files, no middlewares are applied.
+ */
+
+use Illuminate\Support\Facades\Route;
+use JustBetter\GlideDirective\Controllers\ImageController;
+
+$patterns = [
+    'file' => '.*',
+    'format' => '\..+',
+];
+
+Route::get(
+    config('justbetter.glide-directive.cache_prefix', 'img').'/'.config('justbetter.glide-directive.storage_prefix', 'glide-image').'/{preset}/{fit}/{s}/{file}{format}',
```

Comment:
I don't think the default `img` is needed as it's hardcoded in the config. Same for the storage prefix